VideoWeb/VideoWeb/ClientApp/src/app/home/navigator/navigator.component.ts

```diff
@@ -5,28 +5,42 @@ import { DeviceTypeService } from '../../services/device-type.service';
 import { ErrorService } from '../../services/error.service';
 import { pageUrls } from '../../shared/page-url.constants';
 import { UserProfileResponse, Role } from '../../services/clients/api-client';
+import { ConfigService } from 'src/app/services/api/config.service';
 
 @Component({
     selector: 'app-navigator',
     templateUrl: './navigator.component.html'
 })
 export class NavigatorComponent implements OnInit {
+    private enableIOSSupport: boolean;
+    private enableAndroidSupport: boolean;
+
     constructor(
         private router: Router,
         private profileService: ProfileService,
         private errorService: ErrorService,
-        private deviceTypeService: DeviceTypeService
+        private deviceTypeService: DeviceTypeService,
+        private configService: ConfigService
     ) {}
 
     ngOnInit() {
-        if (this.deviceTypeService.isDesktop() || this.deviceTypeService.isIpad()) {
-            this.profileService
-                .getUserProfile()
-                .then(profile => this.navigateToHearingList(profile))
-                .catch(error => this.errorService.handleApiError(error));
-        } else {
-            this.router.navigate([pageUrls.UnsupportedDevice]);
-        }
+        this.configService.getClientSettings().subscribe(settings => {
```

Comment:
Shouldn't we unsubscribe from this on destroy?

Reply:
Oh is this an API call that will resolve itself? I would prefer to be more specific here but I'll approve if it is.